studio/schemas/documents/event.ts

```diff
@@ -106,13 +101,19 @@ export default {
       name: 'content',
       title: 'Content',
       type: 'array',
-      of: [blockContentType,
-       basicIframe],
+      of: [blockContentType, basicIframe],
     },
     {
       title: 'IFrame (Deprecated)',
       name: 'iframe',
       type: 'basicIframe',
```

Comment:
can you also add options: collapsible and collapsable here? :)

Reply:
Adding collapsible and collapsed true to basicIframe does not work.